app/views/forms/odk/_instances.xml.erb

```diff
@@ -11,9 +11,14 @@
   <% end %>
 </instance>
 
+<%# Instances for first level option sets if needed for dynamic calculations %>
+<% @form.option_sets_for_instances.each do |option_set| %>
+  <%= ODK::DecoratorFactory.decorate(option_set).option_instances_for_dynamic_calcs %>
+<% end %>
```

Comment:
The structure of this block and the block below differ even though they seem to be doing very similar things, and I wonder if we can make them more consistent for readability.

Can we assign a new `@option_sets_for_instances` in the same way that `@option_sets` (used below) gets assigned? And if it gets decorated like that one too, we'd just do `option_set.option_instances_for_dynamic_calcs`

Reply:
cool, done!